src/loaders/stm_load.c

```diff
@@ -71,15 +95,17 @@ static int stm_test(HIO_HANDLE * f, char *t, const int start)
 	hio_seek(f, start + 20, SEEK_SET);
 	if (hio_read(buf, 1, 8, f) < 8)
 		return -1;
-	if (memcmp(buf, "!Scream!", 8) && memcmp(buf, "BMOD2STM", 8) && memcmp(buf, "WUZAMOD!", 8))
-		return -1;
 
-	hio_read8(f);
+	if (hio_read8(f) != 0x1a)
```

Comment:
I found that ST2 ignores this, ST3 doesn't. putup11.stm from ModLand has 0x02 instead of 0x1A here.

Reply:
Yes, ST2 ignores this as you can see in st2play, but for now this is the simplest method to probe format. Where did you get putup11.stm with 0x02 instead of 0x1a? Mine copy from SWave Pro distribution has correct header. It looks like patched manually.

Reply:
From ModLand: https://www.exotica.org.uk/mediawiki/index.php?title=Special%3AModland&md=search&mod=putup11&aut=&for=&col=&ins=

Reply:
Thank you. Looks very strange, putup10.stm from Modland is Putup11.stm from Swave Pro distribution and vice versa. The files absolutely identical except of 0x1a byte after tracker "signature". Seems like someone has patched files with unknown reason.

Reply:
I think the file on ModLand was simply a result of a broken FTP upload in the dark ages of file transfer errors. If you could point me at the original file with the correct 0x1A byte, that would be appreciated, because then I can tell ModLand admins to exchange the file.

Reply:
Soundwave Pro: http://old-dos.ru/index.php?page=files&mode=files&do=show&id=5414
Putup songs: https://files.fm/u/4mtj3w2f